src/shared/errors/throw-if-icon-html-missing.ts

```diff
@@ -0,0 +1,12 @@
+import { Icon, IconLookup } from '@fortawesome/fontawesome';
+
+/**
+ * Throws error if icon html missing although there is icon spec.
+ * @param {Icon} iconObj
+ * @param {IconLookup} iconSpec
+ */
+export const faThrowIfIconHtmlMissing = (iconObj: Icon, iconSpec: IconLookup) => {
```

Comment:
Could we change this to 'warn' instead of 'throw', since it's only logging to the console and not throwing an Error/Exception. I.e., it's not actually `throw`-ing anything.

So rename this one `faWarnIfIconHtmlMissing`.